common/domain/replicationTaskExecutor_integration_test.go

```diff
@@ -0,0 +1,864 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
let's rename the files with snake_case

Reply:
it is filename_test.
If we change even the file name to snake case, it will be inconsistent throughout.